bluebrain/repo-bluebrain/packages/py-atlas-direction-vectors/package.py

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyAtlasDirectionVectors(PythonPackage):
+    """Commands to compute direction vectors in volumetric brain regions."""
+    homepage = "https://github.com/BlueBrain/atlas-direction-vectors"
+    git      = "git@github.com/BlueBrain/atlas-direction-vectors.git"
+    pypi     = "atlas-direction-vectors/atlas-direction-vectors-0.1.2.tar.gz"
+
+    version('0.1.2', tag='d87ff0caa7e03f62fa03fcd3230c398c3f74f5b7f550ac204098befbba6b8f45')
+
+    depends_on('py-setuptools', type=('build', 'run'))
+    depends_on('py-atlas-commons@0.1.4:', type=('build', 'run'))
+    depends_on('py-setuptools-scm', type='build')
+    depends_on('py-click@7.0:', type=('build', 'run'))
+    depends_on('py-numba', type=('build', 'run'))
+    depends_on('py-numpy@1.15.0:', type=('build', 'run'))
+    depends_on('py-numpy-quaternion', type=('build', 'run'))
```

Comment:
In setup.py it's `numpy-quaternion[numba]>=2021.11.4.15.26.3` but this version isn't available in spack.
I see the patch below "Purge version constraints caused by old (outdated) numba incompatibilities", but I just wonder why we can depend on an older version on spack (2020.11.2.17.0.49)?

Reply:
Not sure, I'll try and get this built, and look into that after; thanks.